.circleci/config/jobs/@docker.yml

```diff
@@ -13,6 +13,14 @@ docker-image:
         steps:
           - aeplugin_install_devmode
     - setup_docker
+    - when:
```

Comment:
I'm not sure this is best approach as the following condition will also match

Reply:
so what is your proposition, as my this pipelien understanding it's best place...

Reply:
ok now it should  match when required

Reply:
What I mean is that when this job is invoked from:
- docker-image-latest (tag: latest) AND
- docker-image-latest-bundle (tag: latest-bundle)

The following step that matches for tag: `^latest.*$` will be run twice, although it would do nothing wrong it's definitely wrong and unwanted behaviour.